planning/behavior_velocity_run_out_module/src/debug.hpp

```diff
@@ -35,14 +35,17 @@ class DebugValues
 public:
   enum class TYPE {
     CALCULATION_TIME = 0,
-    CALCULATION_TIME_COLLISION_CHECK = 1,
-    LATERAL_DIST = 2,
-    LONGITUDINAL_DIST_OBSTACLE = 3,
-    LONGITUDINAL_DIST_COLLISION = 4,
-    COLLISION_POS_FROM_EGO_FRONT = 5,
-    STOP_DISTANCE = 6,
-    NUM_OBSTACLES = 7,
-    LATERAL_PASS_DIST = 8,
+    CALCULATION_TIME_PATH_PROCESSING,
```

Comment:
Why did you remove the number? (`= 1`, `= 2`...)
I think it's useful when debugging with tools like plotjuggler.

Reply:
https://github.com/autowarefoundation/autoware.universe/pull/5885/commits/eb1fb4eff91bbb06a5d8ec886e8b3d9bca64f794
DONE